src/commandHandlers/issue.ts

```diff
@@ -33,28 +28,47 @@ export const command = async (arg: string, embed: MessageEmbed, message: Message
         // await repository name
         let repository = await message.channel.awaitMessages((m: Message) => m.author.id === message.author.id, { max: 1, time: 10000, errors: ['time'] }).then(collected => {
             const res = collected.first();
-            if(res) return res.content;
-            else return 'EddieBot';
+            if(res) {
+                return res.content;
+            }
+            return 'EddieBot';
```

Comment:
What do you think of changing to a ternary here @itsphaze ?
```suggestion
            return res ? res.content : 'EddieBot';
```

Reply:
It's the same for me, I'll go with your suggestion!